src/ChirpInfrastructure/CheepService.cs

```diff
@@ -71,4 +77,40 @@ public async Task<List<CheepDTO>> GetCheepsFromAuthorAsync(string author, int pa
 
         return _cheeps;
     }
+    public async Task<int> CreateCheepAsync(int userId, string userName, string text)
+    {
+        Console.WriteLine($"Creating cheep for user {userId} with text: {text}");
```

Comment:
Delete this later

Reply:
Keeping this for now, to make error handling easier if we need it later (this method is also not called at the moment)